### DISCUSSION TITLE
Does SeisSol support vector force?

### DISCUSSION THREAD

**Discussion by buzhiqiuyuebai** [SeisSol/SeisSol#888] [Q&A] [answered]
The point source given in the example is given in the form of moment tensor, does it support the input form of vector force?

**Comment by sebwolf-de** [comment, marked as answer]
Yes: https://seissol.readthedocs.io/en/latest/point-source-older-implementation.html

**Comment by buzhiqiuyuebai** [comment]
> Yes: https://seissol.readthedocs.io/en/latest/point-source-older-implementation.html

Is it enough to change just these[ two lines](https://seissol.readthedocs.io/en/latest/point-source-older-implementation.html#:~:text=header%20line%20(optional%2C%20but%20has%20to%20contain%20%27velocity%27%20to%20be%20recognized)%0Ad1%20d2%20d3%20%20%20%20(optional))? Is it enough to keep the rest of the lines the same?
![image](https://github.com/SeisSol/SeisSol/assets/114474894/f2db664d-d67c-412a-b90e-5ae34dedf918)

**Reply by sebwolf-de** [reply]
Yes.

**Reply by buzhiqiuyuebai** [reply]
Thank you!
